demo/server/src/modules/keycloak/index.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path="./keycloak.d.ts" />
+
+import * as express from 'express';
+import * as session from 'express-session';
+import Keycloak = require('keycloak-connect');
```

Comment:
@TommyJ1994 @paolobueno that works! :) We can change it to that instead of require. :+1: